### PR TITLE
Split tests for setting session description into multiple files (rebase only) 

### PR DESCRIPTION
**Pull request by rwaldron**
@bobholt This is just a rebasing of a PR that I previously reviewed and approved. I didn't want to block @soareschen anymore so I offered to resolve the conflict myself—which this PR is the result of.

### CODE REVIEW

File: webrtc/coverage/set-session-description.txt [hunk 1]
```diff
@@ -236,5 +236,8 @@ Coverage Report
 
   Tested        35
   Not Tested    15
+  Tested        30
+  Not Tested    20
+>>>>>>> Update coverage report for set-session-description
```
Comment (bobholt) [inline webrtc/coverage/set-session-description.txt]: Looks like remainder from a merge conflict resolution.

**Review by bobholt — CHANGES_REQUESTED** [review]

File: webrtc/coverage/set-session-description.txt (hunk 1 above)
Comment (rwaldron) [inline webrtc/coverage/set-session-description.txt]: Derp. Will fix that now

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/257797258)

# Firefox (nightly) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/257797259)

# Sauce (safari) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/257797268)

# Chrome (unstable) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/257797270)

# Sauce (MicrosoftEdge) #

**Review by bobholt — APPROVED** [review]

**Comment by rwaldron** [comment]
Thanks @bobholt!

**Comment by soareschen** [comment]
Oops, this undo some of the removals in #6616. I will make another PR to remove the isClosed-then-abort tests again.

**Pull request merged**